unit-tests/py/rspy/timer.py

```diff
@@ -0,0 +1,28 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+import time
+from rspy.stopwatch import Stopwatch
+
+# A timer counting backwards in time(vs forwards in the `stopwatch` class )
+# It supply basic timer API, start, has_expired..
+class Timer:
+
+    _delta = 0
+    _sw = Stopwatch()
+
+    def __init__(self, timeout):
+        self._delta = timeout
+
+    # Start timer
+    def start(self):
+        self._sw.reset()
+
+    # Check if timer time expired
+    def has_expired(self):
+        return self._sw.get_start() + self._delta <= time.perf_counter()
+
+    # Force time expiration
+    def set_expired(self):
+        self._sw.reset(time.perf_counter() - (self._delta + 0.00001));
```

Comment:
Why the `0.00001`? Can also reset to 0, no?

Reply:
As discussed, we want to make sure the has_expired() function return true. the epsilon is for making sure we don't get weird behavior due to float comparation